cmd/updater/systemd-setup.sh

```diff
@@ -1,26 +1,60 @@
 #!/bin/bash
+# shellcheck disable=2086,2164
 #
 # Run this script to set up the systemd service for algod.
-# The arguments are the username and groupname that algod
-# should run as.
+# The arguments are the username and groupname that algod should run as.
+# If wanting to run as a user service, substitute "--user" for the groupname.
+
+set -e
+
+setup_user() {
```

Comment:
There's nothing in common between setup_user() and setup_root(), would this script be clearer as two scripts? systemd-setup.sh and systemd-user-setup.sh ?

Reply:
I'll break these into two scripts.

Reply:
Done and retested.